scrypto/src/component/stubs.rs

```diff
@@ -89,10 +90,10 @@ extern_blueprint_internal! {
     {
         fn register(&mut self);
         fn unregister(&mut self);
-        fn stake_as_owner(&mut self, stake: Bucket) -> Bucket;
-        fn stake(&mut self, stake: Bucket) -> Bucket;
-        fn unstake(&mut self, stake_unit_bucket: Bucket) -> Bucket;
-        fn claim_xrd(&mut self, bucket: Bucket) -> Bucket;
+        fn stake_as_owner(&mut self, stake: FungibleBucket) -> FungibleBucket;
+        fn stake(&mut self, stake: FungibleBucket) -> FungibleBucket;
+        fn unstake(&mut self, stake_unit_bucket: FungibleBucket) -> FungibleBucket;
```

Comment:
I believe that `unstake` and `claim_xrd` are not correct.

* Input to `unstake` is indeed fungible but the output is a non-fungible claim NFT.
* Input to `claim_xrd` is a non-fungible claim NFT and the output is fungible XRD.